corehq/apps/hqwebapp/templates/hqwebapp/base.html

```diff
@@ -137,41 +137,6 @@
           <script src="{% static 'jquery.cookie/jquery.cookie.js' %}"></script>
           <script src="{% static 'jquery.rmi/jquery.rmi.js' %}"></script>
       {% endcompress %}
-      {% if request.use_jquery_ui %}
```

Comment:
This is inside of a `{% if use_js_bundler %}`, so it only applies to non-webpack pages.